coalib/bearlib/__init__.py

```diff
@@ -97,3 +99,41 @@ def wrapping_function(*args, **kwargs):
         return wrapping_function
 
     return _deprecate_decorator
+
+
+def deprecate_bear(bear):
+    """
+    Use this to deprecate a bear. Say we have a bear:
+
+    >>> class SomeBear:
+    ...     def run(*args):
+    ...         print("I'm running!")
+
+    Now, say you have still users that use ``SomeOldBear`` which is doing the
```

Comment:
badly formulated, try
"Supposing someone still tries to run SomeOldBear, you can decorate it using.."